packages/cli/src/commands/export.ts

```diff
@@ -0,0 +1,54 @@
+import { OpenAPIConverter } from '@mockoon/commons-server';
+import { Command, flags } from '@oclif/command';
+import { promises as fs } from 'fs';
+import { parseDataFiles } from '../libs/data';
+
+export default class Export extends Command {
+  public static description = 'Export a mock API';
+  public static examples = [
+    '$ mockoon-cli export --data ~/data.json --output ./output.json',
+    '$ mockoon-cli export --data ~/data.json --output ./output.json --type open-api-v3'
+  ];
+
+  public static flags = {
+    data: flags.string({
+      char: 'd',
+      description: 'Path or URL to your Mockoon data file',
+      required: true
+    }),
+    output: flags.string({
+      char: 'o',
+      description: 'Generated OpenApi v3 path and name (e.g. `./output.json`)',
+      required: true
+    }),
+    type: flags.string({
+      char: 't',
+      description: 'Export type',
+      options: ['open-api-v3'],
+      parse: (input) => input.toLowerCase(),
+      default: 'open-api-v3',
+      required: false
+    })
```

Comment:
I am not sure the export `--type` flag is really necessary as OpenAPI v3 is the only format supported. And I honestly don't think we will support v2 as an export option.

Reply:
I agree with that point. There's no other sort of export on Mockoon.
I'll remove this flag and if needed it can be added easily without any breaking change.